Cargo.toml

```diff
@@ -42,7 +42,7 @@ exclude = ["crate-template"]
 default-members = ["bin/reth"]
 
 [patch.crates-io]
-revm = { git = "https://github.com/bluealloy/revm" }
-revm-primitives = { git = "https://github.com/bluealloy/revm" }
+revm = { git = "https://github.com/onbjerg/revm", branch = "onbjerg/jump-map-u8" }
+revm-primitives = { git = "https://github.com/onbjerg/revm", branch = "onbjerg/jump-map-u8" }
```

Comment:
do u wanna open a PR for that on revm?

Reply:
https://github.com/bluealloy/revm/pull/402